### DISCUSSION TITLE
Stereoscopic 3D effect strength?

### DISCUSSION THREAD

**Discussion by DROIDshield** [jurialmunkey/skin.arctic.horizon.2#955] [Discussion]
![20230708_181717](https://github.com/jurialmunkey/skin.arctic.horizon.2/assets/97023369/26838830-068a-4fdb-86ce-6a09a76fc23f)
@jurialmunkey is this skin 3D effect ready? I have been using another TV app that is ready and setup right on my TV everything looks like you could reach out and touch it. It's is via Kodi... so I know the setting works.

The text states not all skins are 3D ready, so wondered if this skin is and if not, will it be at all?

**Comment by jurialmunkey** [comment]
Nah, not 3d ready. I haven't added any stereoscopic skin code. 

I don't have a 3d tv so it's not something I'd have any ability to judge what it'd look like. I'd effectively be adding it completely blind.

**Reply by DROIDshield** [reply]
I don't think it is actual 3d as such, by the detail in the text for that menu, what it does is just adds depth to the picture so it looks like a 3d image.

If you get five mins, just have a look at that setting and the text below to go with.

I don't have a 3d TV but using that setting at 1 for big screens really makes stuff look real to touch.

**Reply by jurialmunkey** [reply]
Yeah I know how to implement it in the skin. There's a special tag which to indicate how far apart each layer should appear.

The problem is because I don't have a 3D tv I cannot test and so will have no idea what appropriate values are or how it looks after implementing. 

It's not hard to implement but involves quite a lot of work because the value needs to be added to every group control in the skin to define it's "depth". That amount of work is too involved when I can't even see what the result is and test as I'm going -- I wouldn't have a clue whether something needs adjusting because I'd be doing it blind.

**Reply by DROIDshield** [reply]
NP... cheers for clarification anyway. As always.. the skin is still hot as ever.. keep up the great work.
